corehq/apps/domain/dbaccessors.py

```diff
@@ -34,7 +34,8 @@ def get_doc_ids_in_domain_by_type(domain, doc_type, database=None):
     )]
 
 
-def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000, database=None):
+def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000,
```

Comment:
I chose 0 because I figured in the event someone else wants to use this function, i thought 0 was a saner default, since it will return the startkey you give it, as one would expect couch to, unless told otherwise. Happy to change it if you think 1 is a more normal default or odds of reuse are low enough to not care